int16 senderID
int16 receiverID
string variableName
uint8[] value
int16 msgID
uint64 lamportTime
uint8 type


